components/ide/code/leeway.Dockerfile

```diff
@@ -39,7 +41,7 @@ RUN curl -fsSL https://raw.githubusercontent.com/nvm-sh/nvm/v0.38.0/install.sh |
     && npm install -g yarn node-gyp
 ENV PATH $NVM_DIR/versions/node/v$NODE_VERSION/bin:$PATH
 
-ENV GP_CODE_COMMIT 37f3ceefb1c5ca4e492334316c6b65cf7e665a2d
+ENV GP_CODE_COMMIT 06711200f6a3a19af94f91102fd49d921b1145db
```

Comment:
Could you add a comment why the new version is required

Reply:
The share button added to the editor layout is not present in the older version of vscode we were using. The newer one also has UI changes, bug fixes and other optimizations as well.